### DISCUSSION TITLE
How do you return results using pandas?

### DISCUSSION THREAD

**Discussion by muhaha211** [ultralytics/ultralytics#8301] [Q&A]
Even after reviewing the YOLOv8 documentation, it doesn't seem like there's a way to fetch the result using pandas. Is there a method that I'm unaware of? YOLOv5 had one, but I couldn't find it for YOLOv8.

```
model = YOLO("asd.pt")
results = model.predict("a44.png", verbose=True)
print (results[0].boxes.data.tolist())
```
I'm using the above source code instead for now.

**Comment by glenn-jocher** [comment]
Hi there! 👋 You're right, YOLOv8 doesn't have a built-in method to directly return results as a pandas DataFrame, but you can easily convert the results to a DataFrame using pandas. Here's a quick example based on your code:

```python
import pandas as pd
from ultralytics import YOLO

# Load the model and run prediction
model = YOLO("asd.pt")
results = model.predict("a44.png", verbose=True)

# Convert results to a pandas DataFrame
df = pd.DataFrame(results[0].boxes.data.tolist(), columns=['x', 'y', 'width', 'height', 'confidence', 'class'])
print(df)
```

This will give you a DataFrame with the detection results, which you can then manipulate using pandas as needed. Hope this helps! 😊

**Reply by muhaha211** [reply]
this is so cool! thank you very much!

**Reply by glenn-jocher** [reply]
@muhaha211 you're welcome! 😊 We're thrilled that you're enjoying YOLOv8! If you have any questions or need further assistance, feel free to reach out. Happy coding! 🚀
